vktrace/vktrace_replay/vkreplay_main.cpp

```diff
@@ -94,6 +94,13 @@ vktrace_SettingInfo g_settings_info[] = {
      {&replaySettings.compatibilityMode},
      TRUE,
      "Use compatibiltiy mode, i.e. convert memory indices to replay device indices, default is TRUE."},
+    {"p",
+     "Pause",
+     VKTRACE_SETTING_BOOL,
+     {&replaySettings.pause},
+     {&replaySettings.pause},
+     TRUE,
+     "Pause at the start of the replay."},
```

Comment:
This description of the new option should be consistent with the descritpion in the vktrace.md file